src/test/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSparkIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.test.MiniClusterUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Collections;
+
+
+public class ParallelCopyGCSDirectoryIntoHDFSSparkIntegrationTest extends CommandLineProgramTest {
+
+    @Override
+    public String getTestedToolName() {
+        return ParallelCopyGCSDirectoryIntoHDFSSpark.class.getSimpleName();
+    }
+
+    @Test(groups = {"spark", "bucket"})
+    public void testCopyFile() throws Exception {
+        MiniDFSCluster cluster = null;
+        try {
+            final Configuration conf = new Configuration();
+            // set the minicluster to have a very low block size so that we can test transfering a file in chunks without actually needing to move a big file
+            conf.set("dfs.blocksize", "1048576");
+            cluster = MiniClusterUtils.getMiniCluster(conf);
+
+            // copy a multi-block file
+            final Path tempPath = MiniClusterUtils.getTempPath(cluster, "test", "dir");
+            final String gcpInputPath = getGCPTestInputPath() + "large/dbsnp_138.b37.20.21.vcf";
+            String args =
+                    "--inputGCSPath " + gcpInputPath +
+                            " --outputHDFSDirectory " + tempPath;
+            ArgumentsBuilder ab = new ArgumentsBuilder().add(args);
+            IntegrationTestSpec spec = new IntegrationTestSpec(
+                    ab.getString(),
+                    Collections.emptyList());
+            spec.executeTest("testCopyFile-" + args, this);
+
+            final long fileSizeOnGCS = Files.size(IOUtils.getPath(gcpInputPath));
+
+
+            final String hdfsPath = tempPath + "/" + "dbsnp_138.b37.20.21.vcf";
+
+            org.apache.hadoop.fs.Path outputHdfsDirectoryPath = new org.apache.hadoop.fs.Path(tempPath.toUri());
+
+            try(FileSystem fs = outputHdfsDirectoryPath.getFileSystem(conf)) {
+                long chunkSize = ParallelCopyGCSDirectoryIntoHDFSSpark.getChunkSize(fs);
+                Assert.assertTrue(fileSizeOnGCS > chunkSize);
+            }
+
+            Assert.assertEquals(BucketUtils.fileSize(hdfsPath, null),
+                    fileSizeOnGCS);
+
+            final File tempDir = createTempDir("ParallelCopy");
+
+            BucketUtils.copyFile(hdfsPath, null, tempDir + "fileFromHDFS.vcf");
+            Assert.assertEquals(Utils.calculateFileMD5(new File(tempDir + "fileFromHDFS.vcf")), "7e0204f91fdafcd6b13b6907537f3045");
+        } finally {
+            MiniClusterUtils.stopCluster(cluster);
+        }
+    }
+}
```

Comment:
Would be good to add two additional test cases if you have time: one that tests copying an entire directory, and a second that tests copying a file that is smaller than the chunk size. These additional tests can be very simple and use very small files, since you've already tested the case of a "large" file ("large" being relative to the chunk size) in your existing test.